apps/guardian-ui/src/assets/svgs/LightningIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+interface LightningIconProps {
+  color: string;
+}
+
+export const LightningIcon = ({ color }: LightningIconProps) => {
```

Comment:
Icons should be `.svg` files that get transformed by the project's bundler into React components. Note that all the other files in this folder are plain old SVGs. The color is customized using css's `color` property, the SVG should have the fill as `currentColor`.